recipes/pyrad_arm/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pyrad_arm" %}
+{% set version = "0.4.4" %}
+{% set sha256 = "c806405acf1d5ea6f18b6e6363123ea6a3d88070fa39759528108fa594e2fa70" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pyrad_mch/pyrad_mch-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py2k or win]
+  script:
+    - "{{ PYTHON }} -m pip install . --no-deps -vvv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+
+  host:
+    - python
+    - pip
+    - numpy
```

Comment:
Please add numpy below as descibed here: https://conda-forge.org/docs/maintainer/knowledge_base.html?#building-against-numpy

Reply:
@scopatz,

I am not sure that I understand what you want. Do I need to add numpy in the run section? numpy is already a dependency of arm_pyart. I thought it was not needed to put numpy as explicit dependency in such case. Please clarify

Reply:
If you are linking against numpy, which it looks like you are, you need to add the `pin_compatible('numpy')` line in the run sections

Reply:
Got it! I pushed a new commit. I hope everything is fine now